WordPress/Classes/ViewRelated/NUX/EpilogueSegue.swift

```diff
@@ -5,20 +5,29 @@ protocol EpilogueAnimation {
 
 /// Custom animation to allow presented views to appear to come from behind the presenter
 extension EpilogueAnimation where Self: UIStoryboardSegue {
-    func performEpilogue(completion: @escaping () -> ()) {
+    func performEpilogue(completion: @escaping () -> Void) {
         guard let containerView = source.view.superview else {
             return
         }
         let sourceVC = source
         let destinationVC = destination
         let duration = 0.35
 
-        destinationVC.view.frame = sourceVC.view.frame
+        var frame = sourceVC.view.frame
+        // Adjust for the height of the status bar which seems to be ignored during the transition.
+        frame.origin.y += 20
```

Comment:
Instead of hardcoding the height here, it can be gotten from `UIApplication.shared.statusBarFrame.height`